proof/crefine/ARM/Fastpath_C.thy

```diff
@@ -1526,8 +1526,8 @@ lemma cap_reply_cap_ptr_new_np_updateCap_ccorres:
                    limited_and_simps cap_reply_cap_def
                    limited_and_simps1[OF lshift_limited_and, OF limited_and_from_bool]
                    shiftr_over_or_dist word_bw_assocs mask_def shiftl_shiftr3 word_size)
-  apply (cases m ; clarsimp)
-  apply (cases canGrant ; clarsimp)
+  apply (cases m ; clarsimp simp: true_def)
+  apply (cases canGrant ; clarsimp simp: true_def false_def)
```

Comment:
I also wasn't able to remove these unfolds, which were previously implicitly done by `true_def` and `false_def` being added to the simp set in this file. The goal here is
```
to_bool (SCAST(32 signed → 32) true || (from_bool canGrant << 5 >> 4) && 1) ∧
canGrant = to_bool ((SCAST(32 signed → 32) true >> Suc 0) && 1 || from_bool canGrant)
```

Reply:
yeah, this one is fine as well... I don't think there's much point adding rules for what happens if we right-shift `true`